parquet-column/src/main/java/org/apache/parquet/schema/PrimitiveStringifier.java

```diff
@@ -442,6 +445,21 @@ private void appendHex(byte[] array, int offset, int length, StringBuilder build
     }
   };
 
+  static final PrimitiveStringifier WKB_STRINGIFIER = new BinaryStringifierBase("WKB_STRINGIFIER") {
```

Comment:
@wgtmac and @zhangfengcdt, what do you think about updating this to use WKT only if JTS is available, rather than adding JTS to the classpath for all downstream clients? This could just produce a generic string if a geo library isn't available.

Reply:
I believe it’s reasonable to add JTS as a dependency. Its license is compatible with the ASF, and it’s the most widely used geospatial library in the industry. The API is also very stable. [JTS has been around for over 20 years](https://en.wikipedia.org/wiki/JTS_Topology_Suite).

An alternative would be to implement a standalone WKB parser and maintain it within `parquet-java`, but that could introduce significant long-term maintenance overhead for the community.

For context, the Parquet Geo C++ PR included a standalone WKB parser because (1) there isn’t a clean, well-maintained WKB parser in C++, and (2) Dewey had already implemented WKB parsing in C++ multiple times before.